### PR TITLE
Add unclaimed stakes support

### PR DESCRIPTION
**Pull request by willm30**
This PR adds block ingestor support for the Stakes Tab in the Token Activation Popover.

Test with: https://github.com/JoinColony/colonyCDapp/pull/591

### CODE REVIEW

File: src/handlers/motions/motionFinalized/helpers.ts [hunk 1]
```diff
@@ -243,3 +243,34 @@ export const linkPendingColonyMetadataWithColony = async (
...
+
+export const updateColonyUnclaimedStakes = async (
+  colonyAddress: string,
+  motionTxHash: string,
+  motionDatabaseId: string,
+  updatedStakerRewards: StakerReward[],
+): Promise<void> => {
+  const colony = await getColonyFromDB(colonyAddress);
+  if (colony) {
+    const unclaimedMotionStake = {
```
Comment (willm30) [inline src/handlers/motions/motionFinalized/helpers.ts:255]: When the motion is finalized, all stakes are unclaimed. So we just add a new entry to the `motionsWithUnclaimedStakes` array, with the motion being finalized.

File: src/handlers/motions/motionRewardClaimed/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    const motionWithUnclaimedStakes = motionsWithUnclaimedStakes?.find(
+      ({ motionId }) => motionDatabaseId === motionId,
+    );
+
+    if (motionWithUnclaimedStakes) {
+      const unclaimedRewards = updatedStakerRewards.filter(
+        ({ isClaimed }) => !isClaimed,
+      );
+
+      if (unclaimedRewards.length) {
```
Comment (willm30) [inline src/handlers/motions/motionRewardClaimed/helpers.ts:22]: When a reward is claimed, we check to see if there are any unclaimed rewards remaining for the motion. 

If not, we remove the motion from the `motionsWithUnclaimedStakes` array. This is to make user-specific filtering in the client quicker.

If so, we update the motion with the current set of unclaimed rewards.

Comment (jakubcolony) [reply src/handlers/motions/motionRewardClaimed/helpers.ts:22]: Would be nice to have this comment in-code

**Review by jakubcolony — APPROVED** [review]
LGTM, nice work! 
On a side note you might want to rebase with master to include the codegen changes to save you from having to type the operations manually

**Review by ArmandoGraterol — APPROVED** [review]
LGTM 👌

**Pull request merged**